test.py

```diff
@@ -769,7 +769,7 @@ class PythonTest(Test):
 
     def __init__(self, test_no: int, shortname: str, suite) -> None:
         super().__init__(test_no, shortname, suite)
-        self.path = "pytest"
+        self.path = "./test/pytest"
```

Comment:
Each suite has pytest.ini file where you can provide additional pytest options. Maybe update ./test/cql-pytest/pytest.ini & similar files in all suites?

Reply:
@alecco do you know an option for pytest so that it will work as intended?

Reply:
```
[pytest]
pythonpath = src1 src2
```
https://docs.pytest.org/en/7.1.x/reference/reference.html#confval-pythonpath

or it can be passed as environment variable https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH

Reply:
For context, this is the change in Fedora 37: https://fedoraproject.org/wiki/Changes/PythonSafePath#Detailed_Description

Basically script changes `#!/usr/bin/python3` to `#! /usr/bin/python3 -sP` so `/usr/bin` is not in path anymore (?).

Reply:
It might also be worth changing to invoke line to `python -m pytest` (instead of `pytest` directly).

See https://discuss.python.org/t/python-flag-envvar-to-not-put-current-directory-on-sys-path-but-dont-ignore-pythonpath/4235

Reply:
That's what I did. ./test/pytest is `python -m pytest`.